tensorboard/components/tf_imports/BUILD

```diff
@@ -30,6 +30,19 @@ tf_web_library(
     ],
 )
 
+tf_web_library(
+    name = "polymer-externs",
+    srcs = [
+        "polymer-externs.html",
+    ],
+    path = "/polymer",
+    visibility = ["//visibility:public"],
+    exports = ["@org_polymer"],
```

Comment:
Naïve question: If we omit this “exports”, what happens? (I don’t really
know what `exports` does.)

Reply:
No longer needed but to answer question: it re-exports all the symbols in the `@org_polymer` and make it accessible when someone depends on this target.